lib/perl/Genome/Config/AnalysisProject/Command/CompareInstrumentData.pm

```diff
@@ -0,0 +1,119 @@
+package Genome::Config::AnalysisProject::Command::CompareInstrumentData;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use Set::Scalar;
+use Data::Dumper;
+
+class Genome::Config::AnalysisProject::Command::CompareInstrumentData {
+    is => ['Command::V2'],
+    has => [
+        first_analysis_project => {
+            is => 'Genome::Config::AnalysisProject',
+            doc => 'The first Analysis Project to compare Instrument Data.',
+            shell_args_position => 1,
+        },
+        second_analysis_project => {
+            is => 'Genome::Config::AnalysisProject',
+            doc => 'The second Analysis Project to compare Instrument Data.',
+            shell_args_position => 2,
+        },
+    ],
+    has_optional => [
+        list_intersection => {
```

Comment:
Do we expect the user to only ever pick one of these options or do we think they might pick a mix or all of them. If the first is the expected use case we might consider changing this to be a `mode` that can take one of `intersection`, `first_only`, or `second_only`.